src/buildContext.ts

```diff
@@ -72,20 +68,6 @@ const buildCommonContext = (req: express.Request, additionalContext: {}): Contex
   ...additionalContext,
 });
 
-export interface RegularContextParams {
-  req: express.Request;
-  res: express.Response;
-  payload?: unknown;
-  connection?: undefined;
-}
-
-export interface SubscriptionContextParams {
-  req: express.Request;
-  res: express.Response;
-  connection: { context: { req: express.Request } };
-  payload?: unknown;
-}
-
 export interface ExpressContext {
```

Comment:
Can we rename this to something like `ContextParams`? Open to other suggestions. `ExpressContext` sounds misleading imo

Reply:
Fixed